openmmtools/mcmc.py

```diff
@@ -530,16 +530,16 @@ def apply(self, thermodynamic_state, sampler_state):
         self._before_integration(context, thermodynamic_state)
 
         # Run dynamics.
-        timer.start("{}: step()".format(move_name))
+        timer.start("{}: step({})".format(move_name, self.n_steps))
```

Comment:
This is fine for minimalist output, but would it be more clear to have: 

`"{}: Take {} steps".format(...)`

?